src/MongoDB/Server.c

```diff
@@ -391,11 +393,13 @@ PHP_METHOD(Server, isPassive)
 ZEND_BEGIN_ARG_INFO_EX(ai_Server_executeCommand, 0, 0, 2)
 	ZEND_ARG_INFO(0, db)
 	ZEND_ARG_OBJ_INFO(0, command, MongoDB\\Driver\\Command, 0)
+	ZEND_ARG_OBJ_INFO(0, readPreference, MongoDB\\Driver\\ReadPreference, 1)
```

Comment:
Optional parameters should allow `null` for most of our arguments, especially `objects` where it must.
We don't want to be in the situation again where users have to
```
if ($arg2) {
 $obj->method($arg1, $arg2);
} else {
 $obj->method($arg1);
}
```